qp_shogun/sortmerna/__init__.py

```diff
@@ -0,0 +1,18 @@
+from qiita_client import QiitaCommand
+
+from .sortmerna import sortmerna
+
+__all__ = ['sortmerna']
+
+# Defining the Sortmerna command
+req_params = {'input': ('artifact', ['per_sample_FASTQ'])}
+opt_params = {
```

Comment:
Couldn't add in the actual files but can you remove _all_ the .DS_Store from this PR and add it to the .gitignore file?

Reply:
Added

Reply:
Done.